teachertool/src/components/DebouncedTextarea.tsx

```diff
@@ -39,5 +39,5 @@ export const DebouncedTextarea: React.FC<DebouncedTextareaProps> = ({ intervalMs
         timerId.current = setTimeout(sendChange, intervalMs);
     };
 
-    return <Textarea {...props} onChange={onChangeDebounce} />;
+    return <Textarea {...props} autoResize={true} onChange={onChangeDebounce} />
```

Comment:
Since `DebouncedTextareaProps` extends `TextareaProps`, someone can actually set `autoResize` when specifying a `DebouncedTextarea` component (i.e. `<DebouncedTextarea autoResize={true} ... />`), which I think is a good thing.

So instead of hardcoding this to true for all `DebouncedTextAreas`, could we let it pass-through as a part of the `{...props}` and just set `autoResize` to true in the Notes component?